[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:329 



[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:330 Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.5.3    unsupported  02:22:02Z

App                       Version  Status   Scale  Charm                     Channel        Rev  Exposed  Message
failover                           active       2  opensearch                                 2  no       
grafana-agent                      blocked      4  grafana-agent             latest/edge    256  no       Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
main                               active       1  opensearch                                 1  no       
opensearch                         active       1  opensearch                                 0  no       
self-signed-certificates           active       1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload  Agent      Machine  Public address  Ports     Message
failover/0                   active    executing  2        10.149.22.28    9200/tcp  
  grafana-agent/0            blocked   idle                10.149.22.28              Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
failover/1*                  active    executing  3        10.149.22.189   9200/tcp  
  grafana-agent/1*           blocked   idle                10.149.22.189             Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
main/0*                      active    executing  1        10.149.22.183   9200/tcp  (set-password) 
  grafana-agent/2            blocked   executing           10.149.22.183             Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
opensearch/0*                active    executing  4        10.149.22.194   9200/tcp  
  grafana-agent/3            blocked   idle                10.149.22.194             Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
self-signed-certificates/0*  active    idle       0        10.149.22.215             

Machine  State    Address        Inst id        Base          AZ  Message
0        started  10.149.22.215  juju-bf39ed-0  ubuntu@22.04      Running
1        started  10.149.22.183  juju-bf39ed-1  ubuntu@22.04      Running
2        started  10.149.22.28   juju-bf39ed-2  ubuntu@22.04      Running
3        started  10.149.22.189  juju-bf39ed-3  ubuntu@22.04      Running
4        started  10.149.22.194  juju-bf39ed-4  ubuntu@22.04      Running

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:22:02 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-0.d84* -- (10.149.22.183) -- [executing (since: 02:21:59)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:22:14 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-0.d84* -- (10.149.22.183) -- [idle (since: 02:22:05)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:22:25 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-0.d84* -- (10.149.22.183) -- [idle (since: 02:22:05)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:22:36 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-0.d84* -- (10.149.22.183) -- [idle (since: 02:22:05)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:22:48 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-0.d84* -- (10.149.22.183) -- [idle (since: 02:22:05)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:22:59 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-0.d84* -- (10.149.22.183) -- [idle (since: 02:22:05)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:23:11 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: main - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: main active -- message: None
		main-0.d84* -- (10.149.22.183) -- [idle (since: 02:22:05)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:23:22 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: failover - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: failover active -- message: None
		failover-0.435  -- (10.149.22.28) -- [idle (since: 02:22:16)] active: 
		failover-1.435* -- (10.149.22.189) -- [idle (since: 02:22:17)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:23:35 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.710* -- (10.149.22.194) -- [idle (since: 02:22:30)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


02:23:48 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:348 02:23:51 -- Waiting for model: complete.
[32mINFO    [0m pytest_operator.plugin:plugin.py:862 Model status:

Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.5.3    unsupported  02:23:52Z

App                       Version  Status   Scale  Charm                     Channel        Rev  Exposed  Message
failover                           active       2  opensearch                                 2  no       
grafana-agent                      blocked      4  grafana-agent             latest/edge    256  no       Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
main                               active       1  opensearch                                 1  no       
opensearch                         active       1  opensearch                                 0  no       
self-signed-certificates           active       1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload  Agent  Machine  Public address  Ports     Message
failover/0                   active    idle   2        10.149.22.28    9200/tcp  
  grafana-agent/0            blocked   idle            10.149.22.28              Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
failover/1*                  active    idle   3        10.149.22.189   9200/tcp  
  grafana-agent/1*           blocked   idle            10.149.22.189             Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
main/0*                      active    idle   1        10.149.22.183   9200/tcp  
  grafana-agent/2            blocked   idle            10.149.22.183             Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
opensearch/0*                active    idle   4        10.149.22.194   9200/tcp  
  grafana-agent/3            blocked   idle            10.149.22.194             Missing ['grafana-cloud-config']|['grafana-dashboards-provider']|['logging-consumer']|['send-remote-write'] for cos-a...
self-signed-certificates/0*  active    idle   0        10.149.22.215             

Machine  State    Address        Inst id        Base          AZ  Message
0        started  10.149.22.215  juju-bf39ed-0  ubuntu@22.04      Running
1        started  10.149.22.183  juju-bf39ed-1  ubuntu@22.04      Running
2        started  10.149.22.28   juju-bf39ed-2  ubuntu@22.04      Running
3        started  10.149.22.189  juju-bf39ed-3  ubuntu@22.04      Running
4        started  10.149.22.194  juju-bf39ed-4  ubuntu@22.04      Running

[32mINFO    [0m pytest_operator.plugin:plugin.py:868 Juju error logs:

machine-0: 02:14:45 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-0: 02:14:45 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-self-signed-certificates-0: 02:14:45 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-1: 02:15:02 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-1: 02:15:02 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-main-0: 02:15:02 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-3: 02:15:02 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-3: 02:15:02 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-failover-1: 02:15:02 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-4: 02:15:03 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-4: 02:15:03 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-opensearch-0: 02:15:03 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
machine-2: 02:15:04 ERROR juju.worker.dependency "lxd-container-provisioner" manifold worker returned unexpected error: container types not yet available
machine-2: 02:15:04 ERROR juju.worker.dependency "kvm-container-provisioner" manifold worker returned unexpected error: container types not yet available
unit-failover-0: 02:15:04 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-opensearch-0: 02:16:28 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:16:28 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:16:29 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:16:29 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:16:29 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:16:29 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-0: 02:16:29 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-opensearch-0: 02:16:30 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-0: 02:16:30 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-opensearch-0: 02:16:31 ERROR unit.opensearch/0.juju-log certificates:14: [Errno 111] Connection refused
unit-failover-0: 02:16:31 ERROR unit.failover/0.juju-log certificates:13: [Errno 111] Connection refused
unit-failover-0: 02:16:32 ERROR unit.failover/0.juju-log upgrade-version-a:5: [Errno 111] Connection refused
unit-failover-0: 02:16:33 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:16:33 ERROR unit.failover/0.juju-log upgrade-version-a:5: [Errno 111] Connection refused
unit-opensearch-0: 02:16:33 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:16:34 ERROR unit.failover/0.juju-log upgrade-version-a:5: [Errno 111] Connection refused
unit-opensearch-0: 02:16:34 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:16:35 ERROR unit.opensearch/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:16:35 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-opensearch-0: 02:16:36 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:16:36 ERROR unit.failover/0.juju-log certificates:13: [Errno 111] Connection refused
unit-opensearch-0: 02:16:36 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:16:37 ERROR unit.opensearch/0.juju-log upgrade-version-a:8: [Errno 111] Connection refused
unit-failover-0: 02:16:37 ERROR unit.failover/0.juju-log certificates:13: [Errno 111] Connection refused
unit-failover-1: 02:16:38 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:16:38 ERROR unit.opensearch/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:16:38 ERROR unit.failover/0.juju-log node-lock-fallback:4: [Errno 111] Connection refused
unit-failover-1: 02:16:38 ERROR unit.failover/1.juju-log certificates:13: [Errno 111] Connection refused
unit-opensearch-0: 02:16:39 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:16:39 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:16:39 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:16:39 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-0: 02:16:39 ERROR unit.failover/0.juju-log node-lock-fallback:4: [Errno 111] Connection refused
unit-failover-1: 02:16:39 ERROR unit.failover/1.juju-log node-lock-fallback:4: [Errno 111] Connection refused
unit-opensearch-0: 02:16:40 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-0: 02:16:40 ERROR unit.failover/0.juju-log certificates:13: [Errno 111] Connection refused
unit-failover-1: 02:16:40 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-main-0: 02:16:40 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:16:40 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-1: 02:16:41 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-opensearch-0: 02:16:41 ERROR unit.opensearch/0.juju-log certificates:14: [Errno 111] Connection refused
unit-main-0: 02:16:41 ERROR unit.main/0.juju-log certificates:12: [Errno 111] Connection refused
unit-failover-1: 02:16:42 ERROR unit.failover/1.juju-log node-lock-fallback:4: [Errno 111] Connection refused
unit-opensearch-0: 02:16:42 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-main-0: 02:16:42 ERROR unit.main/0.juju-log upgrade-version-a:0: [Errno 111] Connection refused
unit-failover-1: 02:16:42 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-failover-0: 02:16:43 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:16:43 ERROR unit.opensearch/0.juju-log certificates:14: [Errno 111] Connection refused
unit-main-0: 02:16:43 ERROR unit.main/0.juju-log peer-cluster-orchestrator:10: [Errno 111] Connection refused
unit-failover-1: 02:16:43 ERROR unit.failover/1.juju-log certificates:13: [Errno 111] Connection refused
unit-failover-0: 02:16:43 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:16:44 ERROR unit.main/0.juju-log certificates:12: [Errno 111] Connection refused
unit-opensearch-0: 02:16:44 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:16:44 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:16:44 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:16:44 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:16:44 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:16:44 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:16:44 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:16:44 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-opensearch-0: 02:16:45 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:16:45 ERROR unit.failover/0.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-failover-0: 02:16:46 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:16:46 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 02:16:47 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:16:47 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-main-0: 02:16:47 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:16:47 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 02:16:48 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-main-0: 02:16:48 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:16:48 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-0: 02:16:48 ERROR unit.failover/0.juju-log certificates:13: [Errno 111] Connection refused
unit-failover-1: 02:16:48 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-main-0: 02:16:49 ERROR unit.main/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:16:49 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:16:49 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 02:16:49 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:16:50 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-main-0: 02:16:50 ERROR unit.main/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 02:16:50 ERROR unit.main/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 02:16:50 ERROR unit.main/0.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-main-0: 02:16:50 ERROR unit.main/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-failover-1: 02:16:50 ERROR unit.failover/1.juju-log certificates:13: [Errno 111] Connection refused
unit-failover-0: 02:16:50 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-main-0: 02:16:50 ERROR unit.main/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 02:16:50 ERROR unit.main/0.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:16:50 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:16:50 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:16:50 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:16:50 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-main-0: 02:16:50 ERROR unit.main/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 02:16:50 ERROR unit.main/0.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-failover-1: 02:16:51 ERROR unit.failover/1.juju-log certificates:13: [Errno 111] Connection refused
unit-failover-0: 02:16:51 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:16:51 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:16:51 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:16:52 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:16:52 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 02:16:52 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:16:53 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 02:16:53 ERROR unit.failover/1.juju-log upgrade-version-a:5: [Errno 111] Connection refused
unit-main-0: 02:16:53 ERROR unit.main/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 02:16:53 ERROR unit.main/0.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-failover-1: 02:16:54 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 02:16:55 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 02:16:56 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:16:56 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:16:56 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:16:56 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-main-0: 02:16:56 ERROR unit.main/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 02:16:57 ERROR unit.main/0.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-failover-1: 02:16:57 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:16:57 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:16:57 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:16:57 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:16:57 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:16:57 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:16:57 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:16:57 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:16:57 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:16:57 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:16:58 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-opensearch-0: 02:16:58 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-1: 02:16:58 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:16:58 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:16:58 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:16:58 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:16:58 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:16:58 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:16:58 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:16:59 ERROR unit.failover/1.juju-log peer-cluster-orchestrator:11: [Errno 111] Connection refused
unit-opensearch-0: 02:16:59 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:16:59 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:16:59 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:16:59 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-main-0: 02:17:00 ERROR unit.main/0.juju-log opensearch-peers:1: [Errno 111] Connection refused
unit-main-0: 02:17:00 ERROR unit.main/0.juju-log opensearch-peers:1: Cannot connect to the OpenSearch server...
unit-failover-1: 02:17:00 ERROR unit.failover/1.juju-log upgrade-version-a:5: [Errno 111] Connection refused
unit-failover-1: 02:17:01 ERROR unit.failover/1.juju-log upgrade-version-a:5: [Errno 111] Connection refused
unit-failover-0: 02:17:23 ERROR unit.failover/0.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 02:17:23 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 02:17:23 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 02:17:23 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 02:17:24 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 02:17:24 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-opensearch-0: 02:17:24 ERROR unit.opensearch/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:17:24 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 02:17:24 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-opensearch-0: 02:17:25 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:17:25 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:17:25 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:17:25 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-0: 02:17:25 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:17:25 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 02:17:25 ERROR unit.failover/0.juju-log [Errno 111] Connection refused
unit-failover-1: 02:17:25 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-0: 02:17:26 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:26 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:26 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:26 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:27 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:27 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:27 ERROR unit.failover/1.juju-log opensearch-peers:3: Cannot connect to the OpenSearch server...
unit-failover-0: 02:17:27 ERROR unit.failover/0.juju-log opensearch-peers:3: Cannot connect to the OpenSearch server...
unit-failover-1: 02:17:27 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:27 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:27 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:27 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:27 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:27 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:27 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:27 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:27 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:27 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:27 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:27 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:27 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:27 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:27 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:27 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:27 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:27 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:27 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:27 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:27 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:27 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:27 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:28 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:28 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-opensearch-0: 02:17:28 ERROR unit.opensearch/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-0: 02:17:28 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:28 ERROR unit.failover/0.juju-log opensearch-peers:3: Cannot connect to the OpenSearch server...
unit-failover-0: 02:17:28 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:28 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 02:17:28 ERROR unit.failover/1.juju-log Cannot connect to the OpenSearch server...
unit-failover-1: 02:17:28 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 02:17:29 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:17:29 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:17:29 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:17:29 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:17:29 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-1: 02:17:29 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-failover-1: 02:17:29 ERROR unit.failover/1.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:17:30 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-1: 02:17:30 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 02:17:30 ERROR unit.failover/1.juju-log peer-cluster:9: Cannot connect to the OpenSearch server...
unit-failover-1: 02:17:30 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 02:17:30 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 02:17:30 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 02:17:30 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 02:17:30 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 02:17:30 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-opensearch-0: 02:17:30 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:17:30 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:17:30 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:17:30 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-1: 02:17:31 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 02:17:31 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-0: 02:17:31 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-opensearch-0: 02:17:31 ERROR unit.opensearch/0.juju-log peer-cluster:10: [Errno 111] Connection refused
unit-failover-1: 02:17:31 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-failover-1: 02:17:31 ERROR unit.failover/1.juju-log peer-cluster:9: [Errno 111] Connection refused
unit-opensearch-0: 02:17:32 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:17:32 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:17:32 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:17:32 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-0: 02:17:34 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:37 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:40 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:41 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:41 ERROR unit.failover/1.juju-log opensearch-peers:3: Cannot connect to the OpenSearch server...
unit-failover-1: 02:17:41 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:41 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:42 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:42 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:42 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:42 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:42 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:42 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:42 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:42 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:42 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:43 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:43 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:43 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:43 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:43 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-opensearch-0: 02:17:44 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-failover-1: 02:17:44 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:44 ERROR unit.failover/1.juju-log opensearch-peers:3: Cannot connect to the OpenSearch server...
unit-failover-1: 02:17:44 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:44 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-0: 02:17:44 ERROR unit.failover/0.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:44 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:44 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:44 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:44 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-opensearch-0: 02:17:45 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:17:45 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:17:45 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:17:45 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-failover-1: 02:17:45 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:45 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:45 ERROR unit.failover/1.juju-log opensearch-peers:3: Cannot connect to the OpenSearch server...
unit-failover-1: 02:17:46 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:49 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:52 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-failover-1: 02:17:55 ERROR unit.failover/1.juju-log opensearch-peers:3: [Errno 111] Connection refused
unit-opensearch-0: 02:18:00 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 02:18:01 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 02:18:01 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 02:18:02 ERROR unit.opensearch/0.juju-log peer-cluster:11: [Errno 111] Connection refused
unit-opensearch-0: 02:18:03 ERROR unit.opensearch/0.juju-log [Errno 111] Connection refused
unit-opensearch-0: 02:18:04 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:18:04 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:18:04 ERROR unit.opensearch/0.juju-log opensearch-peers:6: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:18:04 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:18:05 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:18:05 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:18:05 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:18:05 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:18:05 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:18:06 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:18:06 ERROR unit.opensearch/0.juju-log opensearch-peers:6: Cannot connect to the OpenSearch server...
unit-opensearch-0: 02:18:06 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:18:09 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:18:12 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-opensearch-0: 02:18:15 ERROR unit.opensearch/0.juju-log opensearch-peers:6: [Errno 111] Connection refused
unit-grafana-agent-1: 02:19:44 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: preparing operation "install ch:amd64/jammy/grafana-agent-256" for grafana-agent/1: failed to download charm "ch:amd64/jammy/grafana-agent-256" from API server: download request with archiveSha256 length 0 not valid
unit-grafana-agent-0: 02:19:44 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: preparing operation "install ch:amd64/jammy/grafana-agent-256" for grafana-agent/0: failed to download charm "ch:amd64/jammy/grafana-agent-256" from API server: download request with archiveSha256 length 0 not valid
unit-grafana-agent-2: 02:19:44 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: preparing operation "install ch:amd64/jammy/grafana-agent-256" for grafana-agent/2: failed to download charm "ch:amd64/jammy/grafana-agent-256" from API server: download request with archiveSha256 length 0 not valid
unit-grafana-agent-3: 02:19:44 ERROR juju.worker.dependency "uniter" manifold worker returned unexpected error: preparing operation "install ch:amd64/jammy/grafana-agent-256" for grafana-agent/3: failed to download charm "ch:amd64/jammy/grafana-agent-256" from API server: download request with archiveSha256 length 0 not valid
unit-grafana-agent-0: 02:19:45 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-grafana-agent-1: 02:19:45 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-grafana-agent-2: 02:19:45 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk
unit-grafana-agent-3: 02:19:45 ERROR juju.worker.meterstatus error running "meter-status-changed": charm missing from disk

[32mINFO    [0m pytest_operator.plugin:plugin.py:947 Forgetting model main...